### PR TITLE
Milestone 1

### PR DESCRIPTION
**Pull request by petudeveloper**
Use bootstrap to creat header and footer sections

### CODE REVIEW

File: styles/main.css [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  fill: white;
+}
+
+.custom-icon-size {
+  font-size: 1.1rem;
+}
+
+@media (max-width: 769px) {
+  .custom-small-bt {
+    font-size: 0.7rem;
```
Comment (Dipeshtwis) [inline styles/main.css:22]: It should be 768px as per standard.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+                        <li><a href="#" class="text-decoration-none text-dark px-1 border-start">Tech &amp; Science</a></li>
+                        <li><a href="#" class="text-decoration-none text-dark px-1 border-start">Culture</a></li>
+                        <li><a href="#" class="text-decoration-none text-dark px-1 border-start">Newsgeek</a></li>
+                        <li><a href="#" class="text-decoration-none text-dark px-1 border-start">Sports</a></li>
+                        <li><a href="#" class="text-decoration-none text-dark px-1 border-start">Health</a></li>
+                        <li><a href="#" class="text-decoration-none text-dark px-1 border-start">The Debate</a></li>
+                        <li><a href="#" class="text-decoration-none text-dark px-1 border-start">Experts</a></li>
+                        <li><a href="#" class="text-decoration-none text-dark px-1 border-start">Vantage</a></li>
+                        <li><a href="#" class="text-decoration-none text-dark px-1">Search  <i class="fas fa-search"></i></a></li>
+                    </ul>
```
Comment (Dipeshtwis) [inline index.html:60]: It should be wrapped in a form with the search field.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
# Status:  ♻️ Changes Required ♻️
 Hi @petumazo  and @mariordgez

It's good to see that you have implemented bootstrap very well, but in order to approve, we have to work a little bit more. I have also left some comments.

## To Highlight

- [x] Your Readme file is already looking great. 😊
- [x] Repo looks Professional.

## For Approval

### Header

- [x] The Newsweek logo should have a link.
- [x] Login and Subscribe should have links. Also, the login should be of button type.
- [x] The Newsweek logo looks bigger in tablet mode. See the image below.

Screenshot (1301)

- [x] There is a lot of space in desktop mode. See the image below.

Screenshot (1302)_LI

### Footer

- [x] The text right to the Newsweek logo is going up. please fix that.

Screenshot (1302)_LI

## Let's fix all the required change, and we will be done.

(Feel free to contact me on slack by the handle of  **deep** if you have any additional questions).
Happy Coding
[Dipesh Kumar](https://deeptwist.tk) | [Twitter](https://twitter.com/97deepeshkumar)

**Review by Elbie-em — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED :yellow_circle:

Hello @petumazo and @mariordgez. Great job applying almost all changes requested from the previous review. However, in order for this milestone to be approved, you must complete all changes requested. 

- [x]  The main navigation should have a form with a search field and not a link as you have illustrated. See the comment by the previous TSE.

## [OPTIONAL]

- [x] In regards to content disposition, font sizing and colors in your footer, ensure it corresponds to the original

**Original**
Screenshot 2021-04-06 at 11 00 45

**Yours**
Screenshot 2021-04-07 at 02 30 46

### Kindly apply these changes so that we can have this approved. You are doing a great job, keep it up :raised_hands:

*As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 1 code review left for this project. If you think that the code review was not fair, you can request a second opinion using this [form](https://airtable.com/shrQAqnBwek5a0O0s).  *

> Elbie Moonga (Part-time TSE)

> Github (@Elbie-em)
> Slack (Elbie Moonga)
> Twitter ([@ElbieEm](https://twitter.com/ElbieEm))
> LinkedIn([Elbie Moonga](https://www.linkedin.com/in/elbiemoonga/))

**Comment by petudeveloper** [comment]
@Dipeshtwis @Elbie-em Thank you so much for your comments and suggestions. They helped us to clean our code and see some missing things we had.

We corrected all your suggestions:

- Add links to The Newsweek logo.
- Use buttons for Login and Subscribe.
- Correct the size of The Newsweek logo in tablet mode.
- Correct the space in desktop mode.
-  Fix the position of the text right to the Newsweek logo.
- Change the size in the CSS to 768px, as per standard.
- Add form with a search field.

Optional:
- We correct font sizing and colors in the footer.

**Pull request merged**